rst_files/types_methods.rst

```diff
@@ -745,7 +672,6 @@ For the distribution ``ϕ`` we'll assign a ``Distribution`` from the `Distributi
     m = AR1(0.9, 1, 1, Beta(5, 5))
 
 
-
 In this call to the constructor we've created an instance of ``AR1`` and bound the name ``m`` to it
 
 We can access the fields of ``m`` using their names and "dotted attribute" notation
```

Comment:
Should probably put something in here eventually about how direct access is a bad idea.